earth_enterprise/src/fusion/autoingest/sysman/khSystemManager.cpp

```diff
@@ -83,6 +84,9 @@ void systemrc_reload(int signum)
   notify(NFY_WARN, "system log level changed to: %s",
           khNotifyLevelToString(static_cast<khNotifyLevel>(logLevel)).c_str());
   setNotifyLevel(static_cast<khNotifyLevel>(logLevel));
+  // triggering XMLPlatformUtils::Terminate() causes intermittent segfailts
```

Comment:
Can be removed or moved to a separate PR.

Reply:
moving over to the wrapper class PR